### PR TITLE
fix(FOROME-1192): linked clear all button in FR with undo-redo

### PR DESCRIPTION
**Pull request by Lapk1n**
https://quantori.atlassian.net/browse/FOROME-1192

### CODE REVIEW

File: src/store/filter/filter.store.ts [hunk 1]
```diff
@@ -132,7 +132,7 @@ export class FilterStore {
     return undefined
   }
 
-  public get conditions(): ReadonlyArray<TCondition> {
+  public get conditions(): TCondition[] {
```
Comment (AndreyUstyumenko) [inline src/store/filter/filter.store.ts]: why did you remove ReadonlyArray?

Comment (Lapk1n) [reply src/store/filter/filter.store.ts]: fixed

**Review by AndreyUstyumenko — APPROVED** [review]

**Review by QSdmitrioul — APPROVED** [review]
